### PR TITLE
config: Add codeowners

### PR DESCRIPTION
**Pull request by MadVikingGod**
The config project currently does not have a codeowner.

I nominate myself to review and manage this project.

### CODE REVIEW

**Review by MrAlias — APPROVED** [review]

**Review by pellared — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -24,6 +24,8 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 
 bridges/prometheus/                                                     @open-telemetry/go-approvers @dashpole
 
+config/                                                                 @open-telemetry/go-approvers @MadVikingGod
```
Comment (pellared) [inline CODEOWNERS:27]: I can be added as well as I think I was reviewing all PRs so far.

```suggestion
config/                                                                 @open-telemetry/go-approvers @MadVikingGod @pellared
```

**Review by dmathieu — APPROVED** [review]

**Review by codeboten — APPROVED** [review]
can i be added to it as well? or is it strictly for approvers/maintainers of this repo?

**Comment by MrAlias** [comment]
> can i be added to it as well? or is it strictly for approvers/maintainers of this repo?

It is not restricted. Adding you here seems appropriate to me :+1:

File: CODEOWNERS [hunk 2]
```diff
@@ -24,6 +24,8 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 
 bridges/prometheus/                                                     @open-telemetry/go-approvers @dashpole
 
+config/                                                                 @open-telemetry/go-approvers @MadVikingGod @pellared
```
Comment (codeboten) [inline CODEOWNERS:27]:
```suggestion
config/                                                                 @open-telemetry/go-approvers @MadVikingGod @pellared @codeboten
```

**Pull request merged**